### PR TITLE
chore: move journey to teachers LESQ-290

### PR DESCRIPTION
**Pull request by JBR90**
## Description

- Move lesson browsing jouney to /teachers 
- Update and move tests from correct page structure
- updates sitemap config
- updates sonar exclusions 

- [x]  I want the teacher browsing journey to exist at /teachers/...
- [x]  I want the [www.thenational.academy](https://www.thenational.academy/) to not be restyled and remain the same until we are ready to launch

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1905--oak-web-application.netlify.thenational.academy
Browsing jouney starts at `/teachers` or `/teachers-2023 `not` /beta/teachers`

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by kimon-satan — APPROVED** [review]
I had to spend a little time working out what I was supposed to do to test this. It would be helpful if you could fill out the info better on the PR. At the very lest copy over the acceptance criteria but ideally give some instructions on how to test.

**Review by jack-skerrett-oak — APPROVED** [review]
We are now at /teachers and /teachers-2023, lets hope those cloudflare rules work!

**Comment by JBR90** [comment]
> I had to spend a little time working out what I was supposed to do to test this. It would be helpful if you could fill out the info better on the PR. At the very lest copy over the acceptance criteria but ideally give some instructions on how to test.

Noted for next time, thanks for the feedback.

File: src/pages/[viewType]/index.tsx [hunk 1]
```diff
@@ -218,7 +218,7 @@ export const getStaticProps: GetStaticProps<HomePageProps> = async (
     context,
     getProps: async () => {
       const isPreviewMode = context.preview === true;
-
+      console.log(context?.params?.viewType);
```
Comment (mantagen) [inline src/pages/[viewType]/index.tsx:221]: just this log in here

**Review by mantagen — APPROVED** [review]

**Pull request merged**